src/abstract/interfaces/IGovernor.sol

```diff
@@ -289,6 +289,9 @@ interface IGovernor is IERC6372, IERC712 {
      */
     function state(uint256 proposalId) external view returns (ProposalState);
 
+    /// @notice Returns the EIP-5805 token contact used for determine voting power and total supplies.
```

Comment:
/// @notice Returns the EIP-5805 token contract used to determine voting power and total supplies.